docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,389 @@
+   secrets:
+     user: testuser
+     password: passwordvalue
+--
+
+`config`:: Defines information for the connector type.
+`hasAuth`::: A boolean that corresponds to *Requires authentication*. If `true`, this connector will require values for `user` and `password` inside the secrets configuration. Defaults to `true`.
+`headers`::: A `record<string, string>` that corresponds to *Headers*.
+`createIncidentUrl`::: A URL string that corresponds to *Create Case URL*.
+`createIncidentMethod`::: A string that corresponds to *Create Case Method*.
+`createIncidentJson`::: A stringified JSON with Mustache variables that corresponds to *Create Case JSON*.
+`createIncidentResponseKey`::: A string from the response body of the create case method that corresponds to the *External Service Id*.
+`getIncidentUrl`::: A URL string with an *External Service Id* Mustache variable that corresponds to *Get Case URL*.
+`getIncidentResponseExternalTitleKey`::: A string from the response body of the get case method that corresponds to the *External Service Title*.
+`viewIncidentUrl`::: A URL string with either the *External Service Id* or *External Service Title* Mustache variable that corresponds to *View Case URL*.
+`updateIncidentUrl`::: A URL string that corresponds to *Update Case URL*.
+`updateIncidentMethod`::: A string that corresponds to *Update Case Method*.
+`updateIncidentJson`::: A stringified JSON with Mustache variables that corresponds to *Update Case JSON*.
+`createCommentUrl`::: A URL string that corresponds to *Create Comment URL*.
+`createCommentMethod`::: A string that corresponds to *Create Comment Method*.
+`createCommentJson`::: A stringified JSON with Mustache variables that corresponds to *Create Comment JSON*.
+
+`secrets`:: Defines sensitive information for the connector type.
+`user`::: A string that corresponds to *User*. Required if `hasAuth` is set to `true`.
+`password`::: A string that corresponds to *Password*. Required if `hasAuth` is set to `true`.
+
+[float]
+[[define-cases-webhook-ui]]
+=== Define connector in {stack-manage-app}
+
+Define Webhook - Case Management connector properties:
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]
+
+Test Webhook - Case Management action parameters:
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-test.gif[Webhook - Case Management params test]
+
+[float]
+[[cases-webhook-action-configuration]]
+=== Action configuration
+
+Webhook - Case Management actions have the following configuration properties:
+
+Title:: A title for the issue, which is used for searching the contents of the knowledge base.
+Description:: The details about the incident.
+Labels:: The labels for the incident.
+Additional comments:: Additional information for the client, such as how to troubleshoot the issue.
+
+
+[float]
+[[cases-webhook-connector-full-example]]
+== Full example with third-party system
```

Comment:
> == Full example with third-party system

IMO everything hereafter is more suited to a blog rather than the docs. These connector docs are intended as reference information (and will gradually be reduced even further as we move the information into the appropriate API reference pages).  So an example like this (especially since it says the preconfigured Jira connector is the preferred choice) seems more like a learning exercise in a blog.

Reply:
my tech lead requested we put an example here. if you want it in a blog, can we publish it at the same time this is published and link it?

Reply:
Yes, we have links to important blogs in the Machine Learning docs, so it is fine to do so here too once that information is published.